src/diagnostics/default_diagnostics.jl

```diff
@@ -5,10 +5,10 @@
 # level interfaces, add them here. Feel free to include extra files.
 
 """
-    default_diagnostics(model, t_start, t_end; output_writer)
+    default_diagnostics(model, t_start, t_end, reference_date; output_writer)
 
 Return a list of `ScheduledDiagnostic`s associated with the given `model` that use
-`output_write` to write to disk. `t_end` is the expected simulation end time and it is used
+`output_write` to write to disk. `t_end, reference_date` is the expected simulation end time and it is used
```

Comment:
`reference_date` is the start date of the simulation?

Reply:
Yes, at the end of the day.